src/main/java/juja/microservices/keepers/dao/KeepersRepository.java

```diff
@@ -16,13 +18,32 @@
  * @author Vadim Dyachenko
  * @author Dmitriy Lyashenko
  * @author Dmitriy Roy
+ * @author Konstantin Sergey
  */
 @Repository
 public class KeepersRepository {
 
+    private final Logger logger = LoggerFactory.getLogger(this.getClass());
+
     @Inject
     private MongoTemplate mongoTemplate;
 
+    public List<String> getDirections(String uuid) {
+        logger.debug("Received get directions by uuid request. Requested uuid: {}", uuid);
+
+        List<Keeper> queryResult = mongoTemplate.find(new Query(
+                Criteria.where("uuid").is(uuid).and("isActive").is(true)), Keeper.class);
+
+        List<String> result = new ArrayList<>();
```

Comment:
Please change the logic. In the repository we just have to get List of Keepers by uuid. Then to return this result to the service. To create the list of directions is the task of the sevice layer. Please be attentive with methods names.